clang/utils/TableGen/SveEmitter.cpp

```diff
@@ -454,6 +458,8 @@ std::string SVEType::builtin_str() const {
   else if (isBFloat()) {
     assert(ElementBitwidth == 16 && "Not a valid BFloat.");
     S += "y";
+  } else if (isMFloat()) {
+    S += "m";
```

Comment:
Perhaps worth adding `assert(ElementBitwidth == 8 && "Not a valid MFloat.")`?

Reply:
Done